src/database/queries/sessionAttendees.js

```diff
@@ -0,0 +1,124 @@
+const SessionAttendee = require('@database/models/index').SessionAttendee
+const { Op } = require('sequelize')
+const SessionEnrollment = require('@database/models/index').SessionEnrollment
+
+exports.create = async (data) => {
+	try {
+		return await SessionAttendee.create(data)
+	} catch (error) {
+		return error
+	}
+}
+
+exports.findOne = async (filter, options = {}) => {
+	try {
+		const res = await SessionAttendee.findOne({
+			where: filter,
+			...options,
+			raw: true,
+		})
+		return res
+	} catch (error) {
+		return error
+	}
+}
+
+exports.updateOne = async (filter, update, options = {}) => {
+	try {
+		const [rowsAffected] = await SessionAttendee.update(update, {
+			where: filter,
+			...options,
+			individualHooks: true, // Pass 'individualHooks: true' option to ensure proper triggering of 'beforeUpdate' hook.
+		})
+
+		if (rowsAffected > 0) {
```

Comment:
We discarded this pattern right? Always return the raw database object and handle it at the service or caller layer